examples/fastify-modules/src/app.ts

```diff
@@ -36,8 +35,7 @@ export const graphqlHandler = (): RouteHandlerMethod => {
 
     reply.status(response.status)
 
-    const nodeStream = Readable.from(response.body)
-    reply.send(nodeStream)
+    reply.send(response.body)
```

Comment:
Why is Readable.from no longer necessary?

Reply:
Because with this PR, Yoga will use Node-fetch which uses `NodeJS.Readable` instead of `ReadableStream` by default.

Reply:
this is a breaking change, then? People upgrading will need to adjust their code?

Reply:
No `Readable.from` accepts `Readable` as well so `Readable.from` will become extra there.